src/orchestration/polling_message_handler.go

```diff
@@ -17,7 +17,7 @@ func (receiver PollingMessageHandler) HandleMessageContents(message azqueue.Dequ
 	sftpHandler, err := sftp.NewSftpHandler()
 	if err != nil {
 		slog.Error("failed to create sftp handler", slog.Any(utils.ErrorKey, err))
-		// Don't return - just because polling is broken for one partner doesn't mean we should take down imports too
```

Comment:
I think when we originally wrote this comment, the `NewSftpHandler` call was happening somewhere else. It actually doesn't make sense here to not-return, hence the change